tests/suites/magma/magma.sh

```diff
@@ -14,7 +14,7 @@ run_deploy_magma() {
 	juju deploy magma-orc8r --overlay "${overlay_path}" --trust --channel=edge
 
 	echo "Check all Magma project components have ACTIVE status"
-	wait_for 34 '[.applications[] | select(."application-status".current == "active")] | length' 1800
+	wait_for 34 '[.applications[] | select(."application-status".current == "active")] | length' 3600
```

Comment:
You should use @basebandit's new wait-for tool `not_idle_applications` (or whatever it is called)

The problem with this is it's very likely an application is added to removed